README.md

```diff
@@ -3,6 +3,43 @@
 # WebScrapperRoBot
 Simple Web scrapper Bot to scrap webpages using Requests, html5lib and Beautifulsoup.
 
+To set up the project and configure environment variables, follow these steps:
+
+Clone the Repository:
+Clone the project's repository from your preferred version control platform (e.g., Git) to your local machine.
+
+Virtual Environment (Optional):
+It's a good practice to create a virtual environment for the project. You can use virtualenv or venv for this purpose.
+
+bash
+Copy code
+python -m venv venv
+source venv/bin/activate  # On Unix/Linux systems
+Install Dependencies:
+Use pip to install the project's dependencies from the requirements.txt file.
+
+bash
+Copy code
```

Comment:
Think you should Use md formatting here , ```Code ```

Reply:
Ok